### PR TITLE
Adjust allclose atol for the flash attention TPU test

### PR DESCRIPTION
**Pull request by JackCaoG**
This should fix the TPU CI Failure, confirmed locally.

### CODE REVIEW

File: test/test_pallas.py [hunk 1]
```diff
@@ -205,7 +205,7 @@ def test_flash_attention_wrapper(self):
 
     o = flash_attention(q, k, v)
     expected_o = self._attention(q, k, v)
-    self.assertTrue(torch.allclose(o.cpu(), expected_o.cpu()))
+    self.assertTrue(torch.allclose(o.cpu(), expected_o.cpu(), atol=1e-07))
```
Comment (alanwaketan) [inline test/test_pallas.py:208]: How about using 1e-4? Just in case.

Comment (JackCaoG) [reply test/test_pallas.py:208]: let's do 1e-5 lol, we also set the precision to highest in the test.

**Review by alanwaketan — APPROVED** [review]
LGTM.

**Comment by JackCaoG** [comment]
TPU CI passed, I am just going to merge.

**Pull request merged**